pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbinstance/alloydbinstance_with_observability/update.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: alloydb.cnrm.cloud.google.com/v1beta1
+kind: AlloyDBInstance
+metadata:
+  name: alloydbinstance-${uniqueId}
+spec:
+  clusterRef:
+    name: alloydbcluster-${uniqueId}
+  availabilityType: REGIONAL
+  instanceTypeRef:
+    name: alloydbcluster-${uniqueId}
+  queryInsightsConfig:
+    queryPlansPerMinute: 5
+    queryStringLength: 1024
+    recordApplicationTags: false
+    recordClientAddress: false
+  observabilityConfig:
+    enabled: true
+    preserveComments: false
```

Comment:
Could you also add a test case for the updates of other fields when enabled is always true?